modules/core/src/v2/coins/eos.ts

```diff
@@ -871,8 +882,52 @@ export class Eos extends BaseCoin {
     return Bluebird.resolve({}).asCallback(callback);
   }
 
-  verifyTransaction(params: VerifyTransactionOptions, callback?: NodeCallback<boolean>): Bluebird<boolean> {
-    return Bluebird.resolve(true).asCallback(callback);
+  /**
+   * Verify that a transaction prebuild complies with the original intention
+   *
+   * @param params
+   * @param params.txParams params used to build the transaction
+   * @param params.txPrebuild the prebuilt transaction
+   * @param callback
+   */
+  verifyTransaction(params: EosVerifyTransactionOptions, callback?: NodeCallback<boolean>): Bluebird<boolean> {
```

Comment:
This shall also include validation for amount and memo fields, as here:
https://github.com/BitGo/bitgo-microservices/blob/develop/packages/wallet-platform/app/controllers/api/v2/coins/eos.js#L679